src/test/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/ODFERestTestCase.kt

```diff
@@ -106,15 +106,19 @@ abstract class ODFERestTestCase : ESRestTestCase() {
         }
     }
 
+    override fun preserveIndicesUponCompletion(): Boolean {
+        return true
+    }
+
     @Throws(IOException::class)
+    @After
```

Comment:
Why was this added back?

Reply:
when I am running test against a cluster with security plugin, if just remove all indices, it will cause problem. We need to keep security system index.

Reply:
Yes, but I meant the tests were rewritten to assume indices were *not* being deleted anymore. So we do not actually want this to run @After every test

Reply:
Whoops, sorry Maxim 😬

Reply:
move these to only ISM security test file